package.json

```diff
@@ -43,7 +43,7 @@
     "mcl-wasm": "^0.4.5",
     "minimist": "^1.2.5",
     "prettier": "^1.19.1",
-    "prettier-plugin-solidity": "^1.0.0-alpha.59",
+    "prettier-plugin-solidity": "1.0.0-beta.10",
```

Comment:
Note when dealing with `alpha/beta/rc` package versions it is generally a good idea to pin to an exact version until the release is stabilized. Similar to libs which have an unstable major version (i.e. `0.x.y`) and may have breaking changes between minor versions.

Reply:
I see, will follow this practice in the future.